src/Common/tests/XunitTraitsDiscoverers/Discoverers/ActiveIssueDiscoverer.cs

```diff
@@ -23,7 +23,7 @@ public class ActiveIssueDiscoverer : ITraitDiscoverer
         public IEnumerable<KeyValuePair<string, string>> GetTraits(IAttributeInfo traitAttribute)
         {
             IEnumerable<object> ctorArgs = traitAttribute.GetConstructorArguments();
-            Contract.Assert(ctorArgs.Count() >= 2);
+            Debug.Assert(ctorArgs.Count() >= 2);
```

Comment:
The message passed to Environment.FailFast gets written to the Windows Application event log.